nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -148,6 +148,7 @@
     summary = "Nullability type error.",
     tags = BugPattern.StandardTags.LIKELY_ERROR,
     severity = WARNING)
+@SuppressWarnings("BugPatternNaming") // remove once we require EP 2.11+
```

Comment:
Ok, I definitely don't see how this violates https://errorprone.info/bugpattern/BugPatternNaming

What am I missing? A bug in EP 2.10?

Reply:
Yeah I think the docs are either wrong or incomplete.  Here is the warning I get without the suppression:
```
nullaway/src/main/java/com/uber/nullaway/NullAway.java:145: warning: [BugPatternNaming] Setting @BugPattern.name to the class name of the check is redundant
@BugPattern(
^
    (see https://errorprone.info/bugpattern/BugPatternNaming)
```